RoomMe/app/src/main/java/com/example/trocket/roomme/getUsersAsync.java

```diff
@@ -34,7 +50,16 @@ public getUsersAsync(HomeActivity home) {
 
     protected void onPostExecute(ArrayList<User> result)
     {
-        home.getUsersResponse(result);
+        switch (selected) {
+            case 1:
+                home.getUsersResponse(result);
+                break;
+            case 2:
+                homeF.getUsersResponse(result);
+                break;
+            default:
+                break;
+        }
```

Comment:
oooh I see. This is unreadable as hell but as long as no one else has to look at this code except for you, it's whatever.